### ISSUE TITLE
Testura website offline

### ISSUE THREAD

**Issue by jeffward01** [Testura/Testura.Code#98]
Hello!

This is rather low priority for now, I just wanted to create a ticket to track that the website: http://www.testura.net/ is offline.

I think it would be great to make a similar wiki to: https://alirezanet.github.io/Gridify/ or https://z4kn4fein.github.io/stashbox/#/getting-started/overview which will allow us to easier create code snippets, and examples for the Wiki.  

This is a long term ticket of course, but I thought it would be helpful to track it

**Comment by MilleBo** [comment]
Yeah I think the site was taken down a couple of years ago so we should probably remove the link until we creat something new on that domain. 

But I think it's a good idea as the github wiki and readme is not that good/optimal. I think I generated the api documentation somewhere as well but I think I stopped generate it a while ago.
